### ISSUE TITLE
Does vsys impact by the log4j 0 day exploit?

### ISSUE THREAD

**Issue by lcgogo** [virtualeconomy/v-systems#307]
https://www.cisa.gov/uscert/ncas/current-activity/2021/12/10/apache-releases-log4j-version-2150-address-critical-rce

**Comment by vipertechofficial** [comment]
I don't think so, it is written in the `scala` programming language instead `log4j` is a module written in `java`, more than that it seems that the modules used by the vsys blockchain doesn't contain the security flaw, because ho, we told me scala "translate itself" to bytecode, "a lower machine language", so they probably not used log4j with it. But hey look, there is even worse than log4j and it is about BIOS: https://www.techradar.com/news/intel-lenovo-and-more-hit-by-major-bios-security-flaws, meanwhile log4j threats is over. Well, blockchain employ many nodes and the BIOS security flaws related threats isn't general it is only like coron4 virus, well annoying.

Look: In anyway, one with another can agree to snapshot the blockchain if they are threatened by the action of those who have opposed vested interest to it.
